src/Omnipay/Stripe/Subscription/Message/SubscriptionRequest.php

```diff
@@ -34,40 +37,35 @@ public function sendData($data)
     {
         $user = $data['user'];
         $invest = $data['invest'];
-
         $project = $invest->getProject();
 
-        $price = $this->stripe->prices->create([
-            'unit_amount' => ($invest->amount + $invest->donate_amount) * 100,
-            'currency' => 'eur',
-            'recurring' => ['interval' => 'month'],
-            'product' => $this->getStripeProduct($invest)->id
-        ]);
+        $customer = $this->getStripeCustomer($user)->id;
+        $metadata = $this->getMetadata($project, $invest, $user);
+
+        $successUrl = sprintf('%s?session_id={CHECKOUT_SESSION_ID}', $this->getRedirectUrl(
+            'invest',
+            $project->id,
+            $invest->id,
+            'complete'
+        ));
 
         $session = $this->stripe->checkout->sessions->create([
-            'customer' => $this->getStripeCustomer($data['user'])->id,
-            'success_url' => sprintf('%s?session_id={CHECKOUT_SESSION_ID}', $this->getRedirectUrl(
-                'invest',
-                $project->id,
-                $invest->id,
-                'complete'
-            )),
-            'cancel_url' => $this->getRedirectUrl(
-                'project',
-                $project->id
-            ),
-            'mode' => 'subscription',
+            'customer' => $customer,
+            'success_url' => $successUrl,
+            'cancel_url' => $this->getRedirectUrl('project', $project->id),
+            'mode' => CheckoutSession::MODE_SUBSCRIPTION,
             'line_items' => [
                 [
-                    'price' => $price->id,
+                    'price' => $this->stripe->prices->create([
+                        'unit_amount' => $invest->amount * 100,
+                        'currency' => 'eur',
```

Comment:
Should we change this for the settings field Currency?

Reply:
Updated this checkout to use the project's currency and the platform tip checkout to use Config currency. This way each checkout matches the payments recipient's (project and platform) currency.